### ISSUE TITLE
Logs filled with raft failures after nomad server leaves cluster

### ISSUE THREAD

**Issue by phinze** [hashicorp/nomad#1026]
### Nomad version

master
### Operating system and Environment details

Linux Ubuntu 14.04
### Issue

We scaled up a nomad cluster server from 3->4 nodes, and then scaled it back down. Now the logs are full of messages like this:

```
2016/04/04 17:03:18 [DEBUG] raft: Failed to contact 10.151.2.194:4647 in 7h49m10.19959811s
2016/04/04 17:03:18 [DEBUG] raft: Failed to contact 10.151.2.194:4647 in 7h49m10.633643133s
2016/04/04 17:03:19 [DEBUG] raft: Failed to contact 10.151.2.194:4647 in 7h49m11.054592655s
```
### Reproduction steps

Get a healthy nomad server cluster, scale it up one node, scale it back down.

**Comment by dadgar** [comment]
@phinze Did you set the servers again or just removed it?

**Comment by phinze** [comment]
@dadgar This was an ASG scale up 3->4, then back down 4->3 of our nomad servers. Our servers have `leave_on_terminate` and `leave_on_interrupt` set.

(There's a small chance the server AMI in question didn't have leave_on_interrupt set yet, so if a cluster behaves correctly given that, then this is a non-issue.)

**Comment by SkyRocknRoll** [comment]
I am facing the same issue. We scaled down 7 node to 5 node nomad servers. I used  `server-force-leave1` still not able remove those node from servers.

**Comment by dadgar** [comment]
@SkyRocknRoll Yeah force-leave only works within a time window of ~72h since after that Serf forgets about the failed node.

Please follow the Manual recovery section here: https://www.consul.io/docs/guides/outage.html

We will eventually mimic the behavior Consul now has

**Comment by SkyRocknRoll** [comment]
@dadgar Thank You :+1:

**Comment by tgross** [comment]
Doing a bit of issue cleanup. Looks like we shipped Autopilot in [0.8.0]https://github.com/hashicorp/nomad/blob/master/CHANGELOG.md#080-april-12-2018), so this should be long-safe to close. 😁
